src/main/java/application/SessionController.java

```diff
@@ -93,10 +100,16 @@ public AnswerQuestionResponseBean answerQuestion(@RequestBody String body){
             AnswerQuestionResponseBean responseBean = mapper.readValue(resp.toString(), AnswerQuestionResponseBean.class);
             return responseBean;
         } catch (JsonParseException e) {
+            System.out.println("ME: " + e);
+            e.printStackTrace();
             log.error(e);
         } catch (JsonMappingException e) {
+            System.out.println("ME: " + e);
+            e.printStackTrace();
             log.error(e);
         } catch (IOException e) {
+            System.out.println("ME: " + e);
+            e.printStackTrace();
             log.error(e);
```

Comment:
If you're already logging why do you need the printlines and the printStackTrace?